app/src/main/java/com/icebreakers/nexxus/NexxusApplication.java

```diff
@@ -13,6 +14,8 @@
 
     public static final String BASE_TAG = "NX: ";
 
+    private static Profile profile = null;
```

Comment:
This is not the best place to keep the object in memory. 
If you really need fast access, you should store it in SharedPreferences. To to weird app lifecycles, this object may be null a lot of times